changelog/261.improvement.md

```diff
@@ -0,0 +1,5 @@
+Added an optional `ignored_intents` parameter in forms.
+
+- In order to use it, please add in your `domain.yml` file the `ignored_intents` parameter with a value/list of values of your choice after the form's name. Please see https://rasa.com/docs/rasa/forms for more information.
+- This parameter will then update/add the `not_intent` key of all individual slots of a form with its specified value (or list of values).
```

Comment:
Can we explain that in a more user-facing way and less coupled to the implementation? E.g.
```suggestion
- all slot mappings of a form won't apply if the intent of a message is specific in `ignored_intents`
```